packages/wrangler/src/create-worker-upload-form.ts

```diff
@@ -82,6 +83,13 @@ export function createWorkerUploadForm(worker: CfWorkerInit): FormData {
 
 	const metadataBindings: WorkerMetadata["bindings"] = [];
 
+	if (bindings.metadata_binding) {
+		metadataBindings.push({
+			name: bindings.metadata_binding,
+			type: "metadata_binding",
```

Comment:
Type should be `metadata`.

Reply:
^, the actual key in the request should be `metadata` not `metadata_binding`

Reply:
Yeah should change it here too then? 
https://github.com/cloudflare/wrangler2/pull/2096/files/d0385d973c5d62c441fbf781a3b17b79135e8e37#diff-0ec779a40744e36b45a9b28dee06739afa8aa2f00bb9388b68556e7fd87ab99eR199